### ISSUE TITLE
new dataset: HPIDS 2016 SARA 12in

### ISSUE THREAD

**Issue by mitchellryant** [TNRIS/tnris.org#898]
wpd for HPIDS 2016 SARA 12in

- [x] Add data to measures spreadsheet @gmullins
- [x] Route Data Shipment [copy data to TNRIS Projects share] @mitchellryant
- [x] Data Processing [renaming file, data organization, coverage map] @mitchellryant
- [x] Quality Assurance @gmullins
- [x] Download Prep [zip, copy to s3 bucket] @mitchellryant
- [x] Move data to production shares @mitchellryant
- [x] Add to Data Download @mitchellryant https://github.com/TNRIS/data-download/issues/81
- [x] This stuff already done for 6-in SARA: Data Publication [Update data catalog sheet, create data catalog images, supplemental reports staging] @gmullins
- [x] Data Story @gmullins
- [x] Final Publication @christoball @gmullins
- [x] Inform TNRIS Staff of data availability @gmullins

**Comment by gmullins** [comment]
Waiting on one missing tile from Surdex. They posted it but it had a bad band 4.

**Comment by gmullins** [comment]
@mitchellryant Tile redelivery complete. Replaced the mosaic. Put the missing tile into the GeoTIFF and JPEG2000 dir's along with the metadata file. Also copied metadata file into Metadata dir. I did not rename any of them.

**Comment by gmullins** [comment]
@christoball Release looks great! Please update these items: 

- Acquisition Areas map is missing

- The first Preview Image is missing the Color Infrared

- 2nd from last Preview Image has wrong caption. Should be: Oil/Gas Footprint in Refugio County 12in SCALE 1:10,000

**Comment by christoball** [comment]
@gmullins yea, actually I didn't mean to just yet, but I forgot that using the phrase 'close #..." will close it too. Either way, all done.
